core/network/src/main/java/com/school_of_company/network/util/ExpoApiHandler.kt

```
@@ -0,0 +1,41 @@
+                400 -> BadRequestException(message = message)
+                401 -> UnauthorizedException(message = message)
+                403 -> ForBiddenException(message = message)
+                404 -> NotFoundException(message = message)
+                409 -> ConflictException(message = message)
+                500, 501, 502, 503 -> ServerException(message = message)
+                else -> OtherHttpException(
+                    message = message,
+                    code = e.code()
+                )
+            }
+        } catch (e: SocketTimeoutException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)